test/linters/i18n_test.js

```diff
@@ -198,6 +198,88 @@ describe("I18nLinter", function() {
       });
     });
 
+    it("reports a success for pluralization keys not present in the default locale but present in others", function() {
+      mockFs({
+        "/path/to/theme": {
+          locales: {
+            "en.default.json": JSON.stringify({
+              products: {
+                product: {
+                  count: {
+                    one: "un",
+                    other: "autre"
```

Comment:
I would have expected these to be in `en`, not `fr`?

Reply:
Good call. I can update that later for sure. I'll make note of that. 👍